src/kubernetes_cluster/mod.rs

```diff
@@ -1,4 +1,4 @@
 // Copyright 2022 VMware, Inc.
 // SPDX-License-Identifier: MIT
-pub mod proof;
+// pub mod proof;
```

Comment:
Do you want to fix the proof lemmas in this PR or later?

Reply:
Later.